IDisposableAnalyzers/build/IDisposableAnalyzers.targets

```diff
@@ -0,0 +1,13 @@
+<Project>
+
+  <PropertyGroup>
+    <OwnershipAttributesPath Condition="'$(OwnershipAttributesPath)' == ''">$(MSBuildThisFileDirectory)OwnershipAttributes$(DefaultLanguageSourceExtension)</OwnershipAttributesPath>
+
+    <IncludeOwnershipAttributes Condition="'$(IncludeOwnershipAttributes)' == '' and Exists($(OwnershipAttributesPath))">true</IncludeOwnershipAttributes>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IncludeOwnershipAttributes)' == 'true'">
+    <Compile Include="$(OwnershipAttributesPath)" Visible="false" />
+  </ItemGroup>
```

Comment:
Resulting behavior:

|       | No `<IncludeOwnershipAttributes>` in project | `true` in project                   | `false` in project |
|-------|----------------------------------------------|-------------------------------------|--------------------|
| C#    | ✔ Provides attributes                        | ✔ Provides attributes               | 🚫 No attributes    |
| VB/F# | 🚫 No attributes                              | ❌ Build failure due to missing file | 🚫 No attributes    |